File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/whileloop.java
import java.util.Scanner;

public class whileloop {

    public static void main(String[] args) {
        try (// Write your code here
                Scanner read = new Scanner(System.in)) {
            int n = read.nextInt();
            int evenS = 0, oddS = 0;
            while (n != 0) {
                if ((n % 10) % 2 == 0)
                    evenS += n % 10;
                else
                    oddS += n % 10;
                n /= 10;
            }
            System.out.println(evenS + " " + oddS);
        }

    }
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_1/While_loop.c
#include <stdio.h>

int main() {
	int a,i=0;
	int so=0, se=0;
	scanf("%d",&a);
	while(a!=0){
		i = a%10;
		if (i%2 == 0){
			se += i;
		}
		else{
			so += i;
		}
		a /=10;
	}
	// Write your code here
	printf("%d %d",se, so);

	
}